specification/columns/commitmentdiscountpurchasedquantity.md

```diff
@@ -0,0 +1,42 @@
+# Commitment Discount Purchased Quantity
+
+Commitment Discount Purchased Quantity is the total quantity of [Commitment Discount Units](#commitmentdiscountunit) that is eligible for amortization from a [*commitment discount's*](#glossary:commitment-discount) purchase over a corresponding [charge period](#glossary:chargeperiod). The CommitmentDiscountPurchaseQuantity column is only applicable to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount).
+
+The CommitmentDiscountPurchasedQuantity column adheres to the following requirements:
+
+* CommitmentDiscountPurchasedQuantity MUST be present in the billing data when the provider supports *commitment discounts*.
+* CommitmentDiscountPurchasedQuantity MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements.
+* When [ChargeCategory](#chargecategory) is "Purchase" and [CommitmentDiscountId](#commitmentdiscountid) is not null, CommitmentDiscountPurchasedQuantity value MUST be the total, positive quantity of units available for a *commitment discount* for the corresponding *charge period*.
+* CommitmentDiscountPurchasedQuantity MAY be negative, 0, or null in cases where [ChargeClass](#chargeclass) is "Correction".
+* CommitmentDiscountPurchasedQuantity MUST be null in all other cases.
+
+In cases where *ChargeCategory* is "Purchase" and *CommitmentDiscountId* is not null, the following applies:
```

Comment:
Line 9 also applies for the same condition thats mentioned here?

Reply:
Simplified to reduce redundancy